core/ext/mod.rs

```diff
@@ -0,0 +1,30 @@
+#[cfg(feature = "uuid")]
+mod uuid;
+#[cfg(feature = "uuid")]
+pub use uuid::{exec_ts_from_uuid7, exec_uuid, exec_uuidblob, exec_uuidstr, UuidFunc};
+
+#[derive(Debug, Clone, PartialEq)]
+pub enum ExtFunc {
+    #[cfg(feature = "uuid")]
+    Uuid(UuidFunc),
+}
+
+impl std::fmt::Display for ExtFunc {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            #[cfg(feature = "uuid")]
+            ExtFunc::Uuid(uuidfn) => write!(f, "{}", uuidfn),
+            _ => write!(f, "unknown"),
+        }
+    }
+}
+
+impl ExtFunc {
+    pub fn resolve_function(name: &str, num_args: usize) -> Result<ExtFunc, ()> {
```

Comment:
I think it's better to use `Option` here, no? Or if you need `Result`, just import `core::Result` and make sure there's a good `LimboError` that you return from here (e.g. `NoSuchFunction`).